financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -128,7 +129,20 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
      * if activity got recreated and the auth flow is still active then the user hit
      * the back button or closed the custom tabs UI, so return result as canceled.
      */
-    internal fun onActivityResult() {
+    internal fun onBrowserActivityResult() {
+        setState {
+            if (authFlowActive && activityRecreated) {
+                copy(viewEffect = FinishWithResult(Canceled))
+            } else this
+        }
+    }
+
+    /**
+     * The native auth flow activity result is equivalent to the browser flow.
+     *
+     * @see [onBrowserActivityResult]
+     */
+    fun onNativeAuthFlowResult() {
```

Comment:
nit: also make this internal

Reply:
https://github.com/stripe/stripe-android/pull/5140/commits/96023aba9c92689beb703b9341ae3df399ea55c0